turbo.json

```diff
@@ -6,7 +6,8 @@
         // Run `build` in workspaces I depend on first
         "^build"
       ],
-      "outputs": ["dist/**", ".next/**"]
+      // Cache all files emitted to the packages's directories
+      "outputs": ["dist/**", ".next/**", "public/_pagefind/**"]
```

Comment:
@87xie this fixes turbo caching, so now when you remove `_pagefind` dir and rerun `pnpm build:all` - `_pagefind` dir will appear from cache